index.html

```diff
@@ -3,14 +3,102 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>css animations</title>
+  <title>Css - Animation</title>
   
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+        <link rel="stylesheet" href="css/search.css">
+        <link rel="stylesheet" href="css/popup.css">
+        <link rel="stylesheet" href="css/cart.css">
+        <link rel="stylesheet" href="css/index.css">
+        <link rel="stylesheet" href="css/submit.css">
+        <link rel="stylesheet" href="css/input.css">
+        <link rel="stylesheet prefetch" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
+
 </head>
 
 <body>
 
-<h1> CSS Animations </h1>
+	<div class="section-one">
+      <div class="search">
+      	<input type="search" class="search-box" placeholder="search" />
+     	 <span class="search-button">
+     	   <span class="search-icon"></span>
+     	 </span>
+	  </div>
+  </div>
+  
+  
+  <div class="">
+    <div class="flex">
+      <div class="share">
+        <a href="#0" class="toggle"><span>SHARE</span></a>
+        <button type="button" class="close">
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+        <div class="content">
+          <div class="text">
+            <ul>
+              <li>Facebook <span>13</span></li>

```

Comment:
i would do them like :
`<li class='social-media>`
            `<a href="#">Facebook</a>`
             `<p class='followers'>13</p>`
`</li>`
 and then in the css i would put the li to be display flex with justify-content : space-between